src/BusyImage.tsx

```diff
@@ -112,17 +120,18 @@ const BusyImage = (props: BusyImageProps) => {
     color: "white",
     fontSize: "2em",
     textShadow: "2px 2px black",
-    width: "8em",
+    maxWidth: "8em",
```

Comment:
Using _maxWidth_ will force the wrap as the renderer will put in a single word before wrapping. I was wondering if we could put the label in a fixed-width transparent box. That might force the wrapping at the correct length, but not have extra background.